sunpy/util/config.py

```diff
@@ -146,3 +150,35 @@ def _get_user_configdir():
     if not _is_writable_dir(configdir):
         raise RuntimeError(f'Could not write to SUNPY_CONFIGDIR="{configdir}"')
     return configdir
+
+
+def copy_default_config(overwrite=False):
+    """
+    Copies the default sunpy config file to the user's config directory.
+
+    Parameters
+    ----------
+    overwrite : `bool`
+        If True, existing config file will be overwritten.
+    """
+    config_filename = 'sunpyrc'
+    config_file = Path(sunpy.__file__).parent / 'data' / config_filename
+    user_config_dir = Path(_get_user_configdir())
+    user_config_file = user_config_dir / config_filename
+
+    if not _is_writable_dir(user_config_dir):
+        raise RuntimeError(f'Could not write to SunPy config directory {user_config_dir}')
+
+    if user_config_file.exists():
+        if overwrite:
+            message = "User config file already exists. " \
+                      "This will be overwritten with a backup written in the same location."
+            warnings.warn(message, SunpyUserWarning)
```

Comment:
Should we be using the logger here instead of `warnings`?

Reply:
no, warnings raised like this are captured by the logger, this is the correct pattern.